orttraining/orttraining/models/bert/main.cc

```diff
@@ -452,12 +454,15 @@ Status ParseArguments(int argc, char* argv[], BertParameters& params, OrtParamet
         "Log severity must be in the range [", static_cast<int>(logging::Severity::kVERBOSE),
         ", ", static_cast<int>(logging::Severity::kFATAL), "].");
     ort_params.vlog_level = flags["ort_vlog_level"].as<int>();
+
+    params.use_invertible_layernorm = flags["use_invertible_layernorm"].as<bool>();
```

Comment:
probably need to add a way to specify this using the pytorch frontend too

Reply:
yes, will add it.